code_annotations/base.py

```diff
@@ -446,6 +449,12 @@ def check_results(self, all_results):
                             current_group, token, annotation['line_number'])
                         )
 
+                # If we have all members, this group is done
```

Comment:
Moved this out here to catch the cases where a group only has one member. Not sure why you'd do that, but it should work.